02_oop_part_1/src/js/interface/ItemComparator.ts

```diff
@@ -0,0 +1,3 @@
+export interface ItemComparator {
```

Comment:
the ItemWeightComparator class must implement the ItemComparator interface, which extends the Comparator interface